feature/system/management_vrf/README.md

```diff
@@ -0,0 +1,85 @@
+# Test plan
+
+## Summary
+
+Test a non default VRF as a VRF for managing the DUT.
+
+## Lab infrastructure
+
+This test plan requires that the DUT is managed by the lab infrastructure in the following way:
+
+![lab_infra](lab_infra.png)
+
+*   All DUT OOB interfaces are L3 with IPv6 addresses connected to lab infrastructure.
+*   Two DUT ports (say **port-11** and **port-12**) are L3 with IPv6 addresses connected to lab infrastructure.
+*   The DUT can be managed at an IPv6 address hosted by a loopback interface, and **via port-11, port-12 or OBB ports**. The loopback interface should host only one IPv6 address. (so that later the test can extract that IPv6 address and move it to another specified loopback interface in a management VRF).
+*   DUT port-1 and port-2 (L3, no vlan) are with IPv6 addresses and are connected to ATE port-1 and port-2, so that the test can exercise/validate some routing protocols (e.g. BGP) without touching lab env infrastructure.
+*   Username used by the test can only be authenticated via TACACS of the lab infra.
+*   If the DUT has multiple CONTROLLER CARD, one single OOB IP should be assigned.
+
+## Test input args
+
+*   `mgmt_vrf_name`: name of the management VRF the test should use. If the lab infrastructure is already using a non default VRF to manage the device, we recommend to specify the name here so that the test reuses the VRF to avoid unnecessary interruptions.
+*   `lab_infra_loopback_name`: before the test starts, this is the name of the loopback interface that hosts the IPv6 management IP. During the test, we will extract the IPv6 management IP from the specified interface, and  put it under a loopback subinterface 100 (say lo0.100) in the management VRF.
+*   `inband_interfaces`: before the tests start, this is a list of linecard interfaces that the lab infrastructure uses to manage the device. During the test, we will ensure it’s in the management VRF.
+
```

Comment:
I do not know other vendors nut ARISTA alredy have:
- mgmtVRF
- loopback100 in mgmtVRF
- 2 x LAG : Port-channel1 and PortChannel2 in mgmtVRF w/ IPv4 and IPv6 addrersses. This LAGs has one member each form 2 distinct LC.

Hence, perhaps, test should learn this configuration form device and retrive: `mgmt_vrf_name`, `loopback_name _in_mgmt_vrf`, `inband_interfaces` (btw it shall be L3 interfaces, hance `interface` name and `subinterface` index).